packages/amplication-client/src/Resource/git/ServiceWizardConfigurationGitSettings.tsx

```diff
@@ -30,17 +31,22 @@ const ServiceWizardConfigurationGitSettings: React.FC<Props> = ({
   // onGitRepositoryDisconnected,
   formik,
 }) => {
-  const { currentWorkspace, currentProjectConfiguration } =
-    useContext(AppContext);
+  const { currentProjectConfiguration } = useContext(AppContext);
   const [isOverride, setIsOverride] = useState<boolean>(
     formik.values.isOverrideGitRepository || false
   );
   const { trackEvent } = useTracking();
   const { gitRepository } = currentProjectConfiguration;
+  const gitProvider = gitRepository?.gitOrganization?.provider;
   const settingsClassName = isOverride
     ? "gitSettingsPanel"
     : "gitSettingsFromProject";
 
+  const gitRepositoryUrlMap = {
```

Comment:
you sure you dont want to make one and import it ?

Reply:
if I make one, it should be a function and getting the organization name and repository name as arguments